src/compiler/diagnosticMessages.json

```diff
@@ -3310,6 +3310,10 @@
         "category": "Error",
         "code": 7034
     },
+    "Try `npm install @types/{0}` if it exists or adding `declare module '{0}'` in a separate file.": {

```

Comment:
I want to see both "declaration file" and ".d.ts" because "typescript declaration file" is more easily searchable and goes straight to the proper handbook topic :)

@sandersn otherwise I like your tweaks. I guess I agree that "to your project" doesn't add anything. I was trying to nod to the fact that you can't just put the .d.ts in the root of your project and expect it to work - it's gotta be somewhere tsc is configured to look. But maybe that's just obvious, and in any case "to your project" doesn't really say that.